_TIAGO_FILES_/SCRIPTS/EUDEMO_powercycle_builder.py

```diff
@@ -0,0 +1,33 @@
+"""
```

Comment:
This whole folder should be removed from the PR, you shouldn't commit it

Reply:
there also looks like there are some changes on tplobo/powercycle_fix that you might have meant to be here

Reply:
I really meant on _your_ github account not our organisation

Reply:
Done!